agent/backend/pktvisor/scrape.go

```diff
@@ -156,9 +157,78 @@ func (p *pktvisorBackend) scrapeDefault() error {
 	return nil
 }
 
+func (p *pktvisorBackend) scrapeOtlp() {
+	policyID := "__all"
+	exeCtx, execCancelF := context.WithCancel(p.ctx)
+	exeCtx = context.WithValue(exeCtx, "policy_name", policyID)
+	exeCtx = context.WithValue(exeCtx, "policy_id", policyID)
+	exeCtx = context.WithValue(exeCtx, "all", true)
+	go func() {
+		defer execCancelF()
+		var err error
+		count := 0
+		for {
+			if p.mqttClient != nil {
+				p.exporter[policyID], err = p.createOtlpMqttExporter(exeCtx, execCancelF)
+				if err != nil {
+					p.logger.Error("failed to create a exporter", zap.Error(err))
+					return
+				}
+				pFactory := otlpreceiver.NewFactory()
+				cfg := pFactory.CreateDefaultConfig()
```

Comment:
I think at least eventually we may need to be able to configure e.g. the port and path, perhaps not in this PR

Reply:
Agreed. I've created a issue for that: https://linear.app/netboxlabs/issue/ORB-709